map_server 1.14.1 and 1.15.0 are failing to export pthread correctly and is breaking downstream packages

There are 3 regressions with the same issue: 
http://build.ros.org/job/Kbin_uX64__turtlebot3_navigation__ubuntu_xenial_amd64__binary/
http://build.ros.org/job/Kbin_uX64__stdr_server__ubuntu_xenial_amd64__binary/
http://build.ros.org/job/Kbin_uX64__multi_map_server__ubuntu_xenial_amd64__binary/

![image](https://user-images.githubusercontent.com/447804/29089036-e138f934-7c2f-11e7-983d-b6770677556d.png)


![image](https://user-images.githubusercontent.com/447804/29088936-a2c8857a-7c2f-11e7-9af2-623672ce2317.png)

Example failure:
http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__multi_map_server__ubuntu_xenial_amd64__binary/33/console
```
00:00:52.988 -- Looking for pthread.h
00:00:53.079 -- Looking for pthread.h - found
00:00:53.082 -- Looking for pthread_create
00:00:53.150 -- Looking for pthread_create - not found
00:00:53.151 -- Looking for pthread_create in pthreads
00:00:53.230 -- Looking for pthread_create in pthreads - not found
00:00:53.231 -- Looking for pthread_create in pthread
00:00:53.321 -- Looking for pthread_create in pthread - found
00:00:53.324 -- Found Threads: TRUE  
00:00:53.325 -- Found gtest sources under '/usr/src/gtest': gtests will be built
00:00:53.327 -- Using Python nosetests: /usr/bin/nosetests-2.7
00:00:53.400 -- catkin 0.7.6
00:00:53.575 CMake Error at /opt/ros/kinetic/share/map_server/cmake/map_serverConfig.cmake:141 (message):
00:00:53.575   Project 'multi_map_server' tried to find library '-lpthread'.  The library
00:00:53.575   is neither a target nor built/installed properly.  Did you compile project
00:00:53.575   'map_server'? Did you find_package() it before the subdirectory containing
00:00:53.575   its code is included?
00:00:53.575 Call Stack (most recent call first):
00:00:53.575   /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:76 (find_package)
00:00:53.575   CMakeLists.txt:4 (find_package)
00:00:53.575 
00:00:53.575 
00:00:53.577 -- Configuring incomplete, errors occurred!
00:00:53.577 See also "/tmp/binarydeb/ros-kinetic-multi-map-server-2.2.2/obj-x86_64-linux-gnu/CMakeFiles/CMakeOutput.log".
00:00:53.577 See also "/tmp/binarydeb/ros-kinetic-multi-map-server-2.2.2/obj-x86_64-linux-gnu/CMakeFiles/CMakeError.log".
00:00:53.581 	cd /tmp/binarydeb/ros-kinetic-multi-map-server-2.2.2
00:00:53.581 	cd obj-x86_64-linux-gnu
00:00:53.581 	"tail -v -n +0 CMakeCache.txt"
```

--- a/map_server/CMakeLists.txt
+++ b/map_server/CMakeLists.txt
@@ -21,12 +21,8 @@ endif()
 catkin_package(
     INCLUDE_DIRS
         include
-        ${SDL_INCLUDE_DIR}
-        ${SDL_IMAGE_INCLUDE_DIRS}
     LIBRARIES
         map_server_image_loader
-        ${SDL_LIBRARY}
-        ${SDL_IMAGE_LIBRARIES}
     CATKIN_DEPENDS
         roscpp
         nav_msgs

